ui/js/models/resultsets_store.js

```diff
@@ -255,14 +255,12 @@ treeherder.factory('ThResultSetStore', [
         };
 
         var getSelectedJob = function(repoName){
-            return { el:repositories[repoName].lastJobElSelected,
+            return { el:{},
                      job:repositories[repoName].lastJobObjSelected };
         };
 
         var setSelectedJob = function(
             repoName, lastJobElSelected, lastJobObjSelected){
-
-            repositories[repoName].lastJobElSelected = lastJobElSelected;
```

Comment:
Likewise, revert this change if you're going to update this element via a selector.